plutus-core-spec/figures/PlutusCoreReduction.tex

```diff
@@ -166,8 +170,28 @@
     
     \caption{CK Machine}
     \label{fig:Plutus_core_ck_machine}
+%%%%%%%%%%%%%
+\vskip5mm
+\red{The machine has states of the form $s \triangleright M$ and $s \triangleleft V$
+where $M$ is a term and $V$ is a value, but there are a number of inconsistencies with the grammar.
+For example:
+\begin{itemize}
+\item There's a transition for the state $s \triangleright \{M A\}$, but according to Figure 3 
+ $\{M A\}$ isn't a term; there is a term $\{M S\}$, but this requires a type value (normalised type?)
+$S$, and $A$ denotes a type.  In fact, all of the transitions listed above involve $A$s, but terms 
+in the grammar all contain $S$s.
+%
+\item If $x$ is a variable name then $x$ is a term, but there's no transition for this.
```

Comment:
Yes, because it's an impossible case as this can only happen if you feed an open term to the CK machine.